### DISCUSSION TITLE
Master Unlocker 1.34.0

### DISCUSSION THREAD

**Discussion by jhowkNx** [jhowkNx/Master-Unlocker#44] [Announcements]
# Download: []

 - Character Statistics Manager
   - Added Longest Single Game Session (Limited)
   - Added Average Time Per Session (Limited)

 - Special Tops
   - Added Open Wheel Sponsor Tee

 - Added Back to Story Mode
 - Added Money Transfer
 - Added option to reset EVERYTHING (except money) - Ranks, stats, and all characters will be wiped/reset to the original
 - Instant Finish MC Business will no longer be a loop
 - Fixed MC Business selling products beyond what it has, causing users to receive an exorbitant amount as payment and then being ignored.
 - Miscellaneous improvements

<hr /><em>This discussion was created from the release <a href='https://github.com/jhowkNx/Master-Unlocker/releases/tag/1.34.0'>Master Unlocker 1.34.0</a>.</em>

**Comment by ghost** [comment]
reset EVERYTHING (except money): does this mean that RSTAR will see us as a NEW player that just started GTA?

In other words, there will be no record of anything on Social Club, like kills, time on GTA Online, Archenemy, favorite weapon, all vehicle, all properties (apartments, nightclub, bunker), all will be gone?  We would be a completely NEW player to RSTAR, right?

**Reply by jhowkNx** [reply]
Yes, except that the money and some properties will not be deleted, but all the progress you've made with them will be reset to the original game. Everything you have will be reset to default.

**Comment by Tweeter83** [comment]
Started having an issue today with the LUA unloading as soon as I try to open it's submenu.  It will load without error, but when opening it's menu it unloads and the script has to be restarted, no error message is listed either.

**Reply by jhowkNx** [reply]
in the latest versions of Master Unlocker, users will no longer be able to access Master Unlocker in story mode, so you will avoid some problems and also add a feature to prevent you from using the script when it is no longer compatible with the current version of GTA Online.
